gradle/validation/dependency-check.gradle

```diff
@@ -0,0 +1,12 @@
+// This adds OWASP vulnerability validation of project dependencies
+
+// This should be false only for debugging.
+def failOnError = true
```

Comment:
It might make sense to set this to false to get this into the gradle-master build and then once merged to master change the default.

I personally find failing the build on OWASP errors can be really frustrating with new CVEs out all the time. I like to see the warnings/errors but not fail the build. This depends on the severity of the CVE as well sometimes.

Reply:
It can be a property. Then you can only warn on default builds but have it fail the build if requested ("-Pvalidation.failOn=owasp,..."). A CI job could run these so that we get a warning.

Reply:
It can be a property. Then you can only warn on default builds but have it fail the build if requested ("-Pvalidation.failOn=owasp,..."). A CI job could run these so that we get a warning.

Reply:
It is now configurable with -Pvalidation.owasp.fail=true (default not fail)